src/main.rs

```diff
@@ -31,6 +31,45 @@ struct UsageError {
     message: String,
```

Comment:
Just because I want to be able to tell whether I'm dealing with a UI error or not from `main`, where all errors are formatted and printed.  Would there be an easier way to do this? I don't think a type alias would work, right?